toolkit/scripts/livepatching/template_livepatch.spec

```diff
@@ -137,11 +139,16 @@ do
     [[ "$patch" == *.patch ]] && cat "$patch" >> $all_patches_file
 done
 
-kpatch-build -ddd \
+if ! kpatch-build -ddd \
     --sourcedir . \
     --vmlinux %{_libdir}/debug/lib/modules/%{kernel_version_release}/vmlinux \
     --name %{livepatch_name} \
     $all_patches_file
+then
+    echo "ERROR: failed to build livepatch module. Logs from '%{kpatch_logs_file}':" >&2
```

Comment:
Do we have to set the `kpatch_logs_file` while we do the `kpatch-build`  or it is the default location

Reply:
I'm using the default location. Basically, for any failures I've seen so far kpatch was telling me that more logs are available under `/root/.kpatch/build.log`, so I decided to just use that.